### DISCUSSION TITLE
Are urls that specify ref sequence aliases supposed to work in JB2 2.0.5?

### DISCUSSION THREAD

**Discussion by jasongallant** [GMOD/jbrowse-components#3249] [General]
Hi @cmdcolin and team!

Curious, I'm trying to get the refname aliases to work on URL request:

This works fine:
https://my.site/jbrowse2/?assembly=${ASSEBMLY_NAME}&loc=${SCAFFOLD_NAME}:1..10 

This works fine:
https://my.site/jbrowse2/?assembly=${ASSEBMLY_ALIAS_NAME}&loc=${SCAFFOLD_NAME}:1..10 

This doesn't seem to work:
https://my.site/jbrowse2/?assembly=${ASSEMBLY_NAME}&loc=${SCAFFOLD_ALIAS_NAME}:1..10 

Jbrowse posts error: "Error: unknown reference sequence name in location "${SCAFFOLD_ALIAS_NAME}:1..10""

I have the "refNameAliases" properly setup, I think, using the GUI configuration tool.  Any thoughts on how to debug, or am I expecting something that shouldn't work?

Thanks!

Jason

**Comment by jasongallant** [comment]
UGH.  It was a line-ending format issue.  Sorry!

**Reply by cmdcolin** [reply]
no worries! glad that you found a fix. we could consider handling different e.g. windows line endings in some cases, i think a lot of code does assume just unix line endings though. feel free to let us know about any other issues you run into, glad to hear from you as well :)
